xyzOrthographicProjection.c

```diff
@@ -0,0 +1,89 @@
+#include xyzOrthographicProjection.h
+

```

Comment:
all of these files should be in your corresponding folder renamed to your functionality